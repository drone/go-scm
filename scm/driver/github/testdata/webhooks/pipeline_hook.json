{
  "action": "in_progress",
  "workflow_run": {
    "id": 14030131450,
    "name": "CI",
    "node_id": "WFR_kwLOE_pLVc8AAAADRELQ-g",
    "head_branch": "shivamnegi94-patch-27",
    "head_sha": "f4229db4b70a8dadeb687f5ccce5fd327c04461a",
    "path": ".github/workflows/blank.yml",
    "display_title": "Update NewSre_v1.yaml",
    "run_number": 2,
    "event": "pull_request",
    "status": "completed",
    "conclusion": "success",
    "workflow_id": 151617408,
    "check_suite_id": 36111489428,
    "check_suite_node_id": "CS_kwDOE_pLVc8AAAAIaGmZlA",
    "url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/runs/14030131450",
    "html_url": "https://github.com/shivamnegi94/testrepo/actions/runs/14030131450",
    "pull_requests": [
      {
        "url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls/164",
        "id": 2412967773,
        "number": 164,
        "head": {
          "ref": "shivamnegi94-patch-27",
          "sha": "f4229db4b70a8dadeb687f5ccce5fd327c04461a",
          "repo": {
            "id": 335170389,
            "url": "https://api.github.com/repos/shivamnegi94/testrepo",
            "name": "testrepo"
          }
        },
        "base": {
          "ref": "main",
          "sha": "ded75dda298abfbf9f1900092e92b43f23bdd6d2",
          "repo": {
            "id": 335170389,
            "url": "https://api.github.com/repos/shivamnegi94/testrepo",
            "name": "testrepo"
          }
        }
      }
    ],
    "created_at": "2025-03-24T08:03:42Z",
    "updated_at": "2025-03-24T08:03:46Z",
    "actor": {
      "login": "shivamnegi94",
      "id": 58415634,
      "node_id": "MDQ6VXNlcjU4NDE1NjM0",
      "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shivamnegi94",
      "html_url": "https://github.com/shivamnegi94",
      "followers_url": "https://api.github.com/users/shivamnegi94/followers",
      "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
      "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
      "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
      "repos_url": "https://api.github.com/users/shivamnegi94/repos",
      "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
      "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
      "type": "User",
      "user_view_type": "public",
      "site_admin": false
    },
    "run_attempt": 1,
    "referenced_workflows": [

    ],
    "run_started_at": "2025-03-24T08:03:42Z",
    "triggering_actor": {
      "login": "shivamnegi94",
      "id": 58415634,
      "node_id": "MDQ6VXNlcjU4NDE1NjM0",
      "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shivamnegi94",
      "html_url": "https://github.com/shivamnegi94",
      "followers_url": "https://api.github.com/users/shivamnegi94/followers",
      "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
      "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
      "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
      "repos_url": "https://api.github.com/users/shivamnegi94/repos",
      "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
      "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
      "type": "User",
      "user_view_type": "public",
      "site_admin": false
    },
    "jobs_url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/runs/14030131450/jobs",
    "logs_url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/runs/14030131450/logs",
    "check_suite_url": "https://api.github.com/repos/shivamnegi94/testrepo/check-suites/36111489428",
    "artifacts_url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/runs/14030131450/artifacts",
    "cancel_url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/runs/14030131450/cancel",
    "rerun_url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/runs/14030131450/rerun",
    "previous_attempt_url": null,
    "workflow_url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/workflows/151617408",
    "head_commit": {
      "id": "f4229db4b70a8dadeb687f5ccce5fd327c04461a",
      "tree_id": "36698bb567243f3d104f398e3b6e1a339570d3dc",
      "message": "Update NewSre_v1.yaml",
      "timestamp": "2025-03-24T08:03:33Z",
      "author": {
        "name": "Shivam Negi",
        "email": "shivam.negi@harness.io"
      },
      "committer": {
        "name": "GitHub",
        "email": "noreply@github.com"
      }
    },
    "repository": {
      "id": 335170389,
      "node_id": "MDEwOlJlcG9zaXRvcnkzMzUxNzAzODk=",
      "name": "testrepo",
      "full_name": "shivamnegi94/testrepo",
      "private": false,
      "owner": {
        "login": "shivamnegi94",
        "id": 58415634,
        "node_id": "MDQ6VXNlcjU4NDE1NjM0",
        "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/shivamnegi94",
        "html_url": "https://github.com/shivamnegi94",
        "followers_url": "https://api.github.com/users/shivamnegi94/followers",
        "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
        "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
        "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
        "repos_url": "https://api.github.com/users/shivamnegi94/repos",
        "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
        "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "html_url": "https://github.com/shivamnegi94/testrepo",
      "description": null,
      "fork": false,
      "url": "https://api.github.com/repos/shivamnegi94/testrepo",
      "forks_url": "https://api.github.com/repos/shivamnegi94/testrepo/forks",
      "keys_url": "https://api.github.com/repos/shivamnegi94/testrepo/keys{/key_id}",
      "collaborators_url": "https://api.github.com/repos/shivamnegi94/testrepo/collaborators{/collaborator}",
      "teams_url": "https://api.github.com/repos/shivamnegi94/testrepo/teams",
      "hooks_url": "https://api.github.com/repos/shivamnegi94/testrepo/hooks",
      "issue_events_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/events{/number}",
      "events_url": "https://api.github.com/repos/shivamnegi94/testrepo/events",
      "assignees_url": "https://api.github.com/repos/shivamnegi94/testrepo/assignees{/user}",
      "branches_url": "https://api.github.com/repos/shivamnegi94/testrepo/branches{/branch}",
      "tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/tags",
      "blobs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/blobs{/sha}",
      "git_tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/tags{/sha}",
      "git_refs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/refs{/sha}",
      "trees_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/trees{/sha}",
      "statuses_url": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/{sha}",
      "languages_url": "https://api.github.com/repos/shivamnegi94/testrepo/languages",
      "stargazers_url": "https://api.github.com/repos/shivamnegi94/testrepo/stargazers",
      "contributors_url": "https://api.github.com/repos/shivamnegi94/testrepo/contributors",
      "subscribers_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscribers",
      "subscription_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscription",
      "commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/commits{/sha}",
      "git_commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/commits{/sha}",
      "comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/comments{/number}",
      "issue_comment_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/comments{/number}",
      "contents_url": "https://api.github.com/repos/shivamnegi94/testrepo/contents/{+path}",
      "compare_url": "https://api.github.com/repos/shivamnegi94/testrepo/compare/{base}...{head}",
      "merges_url": "https://api.github.com/repos/shivamnegi94/testrepo/merges",
      "archive_url": "https://api.github.com/repos/shivamnegi94/testrepo/{archive_format}{/ref}",
      "downloads_url": "https://api.github.com/repos/shivamnegi94/testrepo/downloads",
      "issues_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues{/number}",
      "pulls_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls{/number}",
      "milestones_url": "https://api.github.com/repos/shivamnegi94/testrepo/milestones{/number}",
      "notifications_url": "https://api.github.com/repos/shivamnegi94/testrepo/notifications{?since,all,participating}",
      "labels_url": "https://api.github.com/repos/shivamnegi94/testrepo/labels{/name}",
      "releases_url": "https://api.github.com/repos/shivamnegi94/testrepo/releases{/id}",
      "deployments_url": "https://api.github.com/repos/shivamnegi94/testrepo/deployments"
    },
    "head_repository": {
      "id": 335170389,
      "node_id": "MDEwOlJlcG9zaXRvcnkzMzUxNzAzODk=",
      "name": "testrepo",
      "full_name": "shivamnegi94/testrepo",
      "private": false,
      "owner": {
        "login": "shivamnegi94",
        "id": 58415634,
        "node_id": "MDQ6VXNlcjU4NDE1NjM0",
        "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/shivamnegi94",
        "html_url": "https://github.com/shivamnegi94",
        "followers_url": "https://api.github.com/users/shivamnegi94/followers",
        "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
        "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
        "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
        "repos_url": "https://api.github.com/users/shivamnegi94/repos",
        "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
        "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
        "type": "User",
        "user_view_type": "public",
        "site_admin": false
      },
      "html_url": "https://github.com/shivamnegi94/testrepo",
      "description": null,
      "fork": false,
      "url": "https://api.github.com/repos/shivamnegi94/testrepo",
      "forks_url": "https://api.github.com/repos/shivamnegi94/testrepo/forks",
      "keys_url": "https://api.github.com/repos/shivamnegi94/testrepo/keys{/key_id}",
      "collaborators_url": "https://api.github.com/repos/shivamnegi94/testrepo/collaborators{/collaborator}",
      "teams_url": "https://api.github.com/repos/shivamnegi94/testrepo/teams",
      "hooks_url": "https://api.github.com/repos/shivamnegi94/testrepo/hooks",
      "issue_events_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/events{/number}",
      "events_url": "https://api.github.com/repos/shivamnegi94/testrepo/events",
      "assignees_url": "https://api.github.com/repos/shivamnegi94/testrepo/assignees{/user}",
      "branches_url": "https://api.github.com/repos/shivamnegi94/testrepo/branches{/branch}",
      "tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/tags",
      "blobs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/blobs{/sha}",
      "git_tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/tags{/sha}",
      "git_refs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/refs{/sha}",
      "trees_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/trees{/sha}",
      "statuses_url": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/{sha}",
      "languages_url": "https://api.github.com/repos/shivamnegi94/testrepo/languages",
      "stargazers_url": "https://api.github.com/repos/shivamnegi94/testrepo/stargazers",
      "contributors_url": "https://api.github.com/repos/shivamnegi94/testrepo/contributors",
      "subscribers_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscribers",
      "subscription_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscription",
      "commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/commits{/sha}",
      "git_commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/commits{/sha}",
      "comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/comments{/number}",
      "issue_comment_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/comments{/number}",
      "contents_url": "https://api.github.com/repos/shivamnegi94/testrepo/contents/{+path}",
      "compare_url": "https://api.github.com/repos/shivamnegi94/testrepo/compare/{base}...{head}",
      "merges_url": "https://api.github.com/repos/shivamnegi94/testrepo/merges",
      "archive_url": "https://api.github.com/repos/shivamnegi94/testrepo/{archive_format}{/ref}",
      "downloads_url": "https://api.github.com/repos/shivamnegi94/testrepo/downloads",
      "issues_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues{/number}",
      "pulls_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls{/number}",
      "milestones_url": "https://api.github.com/repos/shivamnegi94/testrepo/milestones{/number}",
      "notifications_url": "https://api.github.com/repos/shivamnegi94/testrepo/notifications{?since,all,participating}",
      "labels_url": "https://api.github.com/repos/shivamnegi94/testrepo/labels{/name}",
      "releases_url": "https://api.github.com/repos/shivamnegi94/testrepo/releases{/id}",
      "deployments_url": "https://api.github.com/repos/shivamnegi94/testrepo/deployments"
    }
  },
  "workflow": {
    "id": 151617408,
    "node_id": "W_kwDOE_pLVc4JCX-A",
    "name": "CI",
    "path": ".github/workflows/blank.yml",
    "state": "active",
    "created_at": "2025-03-24T08:01:34.000Z",
    "updated_at": "2025-03-24T08:01:34.000Z",
    "url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/workflows/151617408",
    "html_url": "https://github.com/shivamnegi94/testrepo/blob/main/.github/workflows/blank.yml",
    "badge_url": "https://github.com/shivamnegi94/testrepo/workflows/CI/badge.svg"
  },
  "repository": {
    "id": 335170389,
    "node_id": "MDEwOlJlcG9zaXRvcnkzMzUxNzAzODk=",
    "name": "testrepo",
    "full_name": "shivamnegi94/testrepo",
    "private": false,
    "owner": {
      "login": "shivamnegi94",
      "id": 58415634,
      "node_id": "MDQ6VXNlcjU4NDE1NjM0",
      "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shivamnegi94",
      "html_url": "https://github.com/shivamnegi94",
      "followers_url": "https://api.github.com/users/shivamnegi94/followers",
      "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
      "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
      "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
      "repos_url": "https://api.github.com/users/shivamnegi94/repos",
      "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
      "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
      "type": "User",
      "user_view_type": "public",
      "site_admin": false
    },
    "html_url": "https://github.com/shivamnegi94/testrepo",
    "description": null,
    "fork": false,
    "url": "https://api.github.com/repos/shivamnegi94/testrepo",
    "forks_url": "https://api.github.com/repos/shivamnegi94/testrepo/forks",
    "keys_url": "https://api.github.com/repos/shivamnegi94/testrepo/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/shivamnegi94/testrepo/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/shivamnegi94/testrepo/teams",
    "hooks_url": "https://api.github.com/repos/shivamnegi94/testrepo/hooks",
    "issue_events_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/events{/number}",
    "events_url": "https://api.github.com/repos/shivamnegi94/testrepo/events",
    "assignees_url": "https://api.github.com/repos/shivamnegi94/testrepo/assignees{/user}",
    "branches_url": "https://api.github.com/repos/shivamnegi94/testrepo/branches{/branch}",
    "tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/tags",
    "blobs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/shivamnegi94/testrepo/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/shivamnegi94/testrepo/languages",
    "stargazers_url": "https://api.github.com/repos/shivamnegi94/testrepo/stargazers",
    "contributors_url": "https://api.github.com/repos/shivamnegi94/testrepo/contributors",
    "subscribers_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscribers",
    "subscription_url": "https://api.github.com/repos/shivamnegi94/testrepo/subscription",
    "commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/shivamnegi94/testrepo/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/shivamnegi94/testrepo/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/shivamnegi94/testrepo/contents/{+path}",
    "compare_url": "https://api.github.com/repos/shivamnegi94/testrepo/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/shivamnegi94/testrepo/merges",
    "archive_url": "https://api.github.com/repos/shivamnegi94/testrepo/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/shivamnegi94/testrepo/downloads",
    "issues_url": "https://api.github.com/repos/shivamnegi94/testrepo/issues{/number}",
    "pulls_url": "https://api.github.com/repos/shivamnegi94/testrepo/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/shivamnegi94/testrepo/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/shivamnegi94/testrepo/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/shivamnegi94/testrepo/labels{/name}",
    "releases_url": "https://api.github.com/repos/shivamnegi94/testrepo/releases{/id}",
    "deployments_url": "https://api.github.com/repos/shivamnegi94/testrepo/deployments",
    "created_at": "2021-02-02T04:41:22Z",
    "updated_at": "2025-03-24T08:01:37Z",
    "pushed_at": "2025-03-24T08:03:33Z",
    "git_url": "git://github.com/shivamnegi94/testrepo.git",
    "ssh_url": "git@github.com:shivamnegi94/testrepo.git",
    "clone_url": "https://github.com/shivamnegi94/testrepo.git",
    "svn_url": "https://github.com/shivamnegi94/testrepo",
    "homepage": null,
    "size": 209,
    "stargazers_count": 0,
    "watchers_count": 0,
    "language": null,
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "has_discussions": false,
    "forks_count": 0,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 56,
    "license": null,
    "allow_forking": true,
    "is_template": false,
    "web_commit_signoff_required": false,
    "topics": [

    ],
    "visibility": "public",
    "forks": 0,
    "open_issues": 56,
    "watchers": 0,
    "default_branch": "main"
  },
  "sender": {
    "login": "shivamnegi94",
    "id": 58415634,
    "node_id": "MDQ6VXNlcjU4NDE1NjM0",
    "avatar_url": "https://avatars.githubusercontent.com/u/58415634?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/shivamnegi94",
    "html_url": "https://github.com/shivamnegi94",
    "followers_url": "https://api.github.com/users/shivamnegi94/followers",
    "following_url": "https://api.github.com/users/shivamnegi94/following{/other_user}",
    "gists_url": "https://api.github.com/users/shivamnegi94/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/shivamnegi94/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/shivamnegi94/subscriptions",
    "organizations_url": "https://api.github.com/users/shivamnegi94/orgs",
    "repos_url": "https://api.github.com/users/shivamnegi94/repos",
    "events_url": "https://api.github.com/users/shivamnegi94/events{/privacy}",
    "received_events_url": "https://api.github.com/users/shivamnegi94/received_events",
    "type": "User",
    "user_view_type": "public",
    "site_admin": false
  }
}