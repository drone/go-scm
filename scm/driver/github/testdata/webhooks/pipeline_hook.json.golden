{
  "Repo": {
    "ID": "335170389",
    "Namespace": "shivamnegi94",
    "Name": "testrepo",
    "Clone": "https://github.com/shivamnegi94/testrepo.git",
    "CloneSSH": "git@github.com:shivamnegi94/testrepo.git",
    "Link": "git://github.com/shivamnegi94/testrepo.git",
    "Branch": "main",
    "Private": false
  },
  "Commit": {
    "Sha": "f4229db4b70a8dadeb687f5ccce5fd327c04461a",
    "Message": "Update NewSre_v1.yaml",
    "Author": {
      "Login": "shivamnegi94",
      "Name": "Shivam Negi",
      "Email": "shivam.negi@harness.io",
      "Avatar": "https://avatars.githubusercontent.com/u/58415634?v=4"
    },
    "Committer": {
      "Login": "shivamnegi94",
      "Name": "Shivam Negi",
      "Email": "shivam.negi@harness.io",
      "Avatar": "https://avatars.githubusercontent.com/u/58415634?v=4"
    },
    "Link": "https://api.github.com/repos/shivamnegi94/testrepo/commits{/sha}"
  },
  "Pipeline": {
    "ID": "14030131450",
    "Status": "in_progress",
    "created_at": "2025-03-24T08:03:42Z",
    "pipeline_url": "https://api.github.com/repos/shivamnegi94/testrepo/actions/runs/14030131450",
    "Branch": "shivamnegi94-patch-27",
    "commit_SHA": "f4229db4b70a8dadeb687f5ccce5fd327c04461a",
    "Author": "Shivam Negi",
    "repo_name": "testrepo"
  },
  "User": {
    "ID": "335170389",
    "Login": "shivamnegi94",
    "Name": "Shivam Negi",
    "Email": "shivam.negi@harness.io",
    "Avatar": "https://avatars.githubusercontent.com/u/58415634?v=4"
  }
}
